shoplist/src/main/java/com/netguru/codereview/network/ShopListRepository.kt

```diff
@@ -0,0 +1,10 @@
+package com.netguru.codereview.network
+
+class ShopListRepository(private val shopListApi: ShopListApi) {
```

Comment:
Create an Interface for shopListRepository, in order to be possible in tests to mock it